apps/measures/models.py

```diff
@@ -10,6 +12,28 @@
 from apps.contrib.translations import TranslatedField
 from apps.home import blocks as apps_blocks
 
+STATUS_CHOICES = [
+    ('1', _('in planning')),
+    ('2', _('running')),
+    ('3', _('finished')),
+]
+
+BEZIRK_CHOICES = [
+    ('1', _('gesamtstädtisch')),
+    ('2', _('Charlottenburg-Wilmersdorf')),
+    ('3', _('Friedrichshain-Kreuzberg')),
+    ('4', _('Lichtenberg')),
+    ('5', _('Marzahn-Hellersdorf')),
+    ('6', _('Mitte')),
+    ('7', _('Neukölln')),
+    ('8', _('Pankow')),
+    ('9', _('Reinickendorf')),
+    ('10', _('Spandau')),
+    ('11', _('Steglitz-Zehlendorf')),
+    ('12', _('Tempelhof-Schöneberg')),
+    ('13', _('Treptow-Köpenick')),
```

Comment:
just a question, why numbers? is it a database thing?

Reply:
no, just random. I think from an example.